test/Network/Minio/XmlParser/Test.hs

```diff
@@ -126,7 +127,7 @@ testParseListObjectsResult = do
               \</ListBucketResult>"
 
     expectedListResult = ListObjectsResult True (Just "opaque") [object1] []
-    object1 = ObjectInfo "my-image.jpg" modifiedTime1 "\"fba9dede5f27731c9771645a39863328\"" 434234
+    object1 = ObjectInfo "my-image.jpg" modifiedTime1 "\"fba9dede5f27731c9771645a39863328\"" 434234 Map.empty
```

Comment:
For the same reason as above comment, we should construct values using `ObjectInfo`s Default instance like,
```hs
object1 = def {
    oiETag = ""
  , oiMetadata = Map.empty
}
```

Reply:
Understood missed it.

Reply:
@harshavardhana ObjectInfo doesn't have an unambiguous Default instance. So, using the ObjectInfo constructor is the only way out for writing unit tests.